lib/libc/gen/ttyname.c

```diff
@@ -1,7 +1,7 @@
 #include <errno.h>
 #include <unistd.h>
 
-const char *ttyname(int fd) {
+char *ttyname(int fd) {
   /* TODO(fzdob): to implement */
```

Comment:
@j-piecuch We would need some support from you to port this function from NetBSD. Namely support for `TIOCPTSNAME`.